tests/snmp/test_snmp_phy_entity.py

```diff
@@ -299,7 +342,10 @@ def test_fan_info(duthosts, enum_rand_one_per_hwsku_hostname, snmp_physical_enti
             parent_entity_info = redis_hgetall(
                 duthost, STATE_DB, PHYSICAL_ENTITY_KEY_TEMPLATE.format(parent_name))
             parent_position = int(parent_entity_info['position_in_parent'])
-            parent_oid = MODULE_TYPE_FAN_DRAWER + parent_position * MODULE_INDEX_MULTIPLE
+            if 'FABRIC-CARD' in parent_name:
+                parent_oid = MODULE_TYPE_FABRIC_CARD + parent_position * MODULE_INDEX_MULTIPLE
```

Comment:
Changes look ok to me, for my understanding, how is the OID computed in sonic-snmp?

Reply:
Ignore the comment, There is SNMP PR associated with this